pylabrobot/resources/resource.py

```diff
@@ -285,6 +285,11 @@ def get_2d_center_offsets(self, n: int = 1) -> List[Coordinate]:
 
     dx = self.get_size_x() / 2
     dy = self.get_size_y() / (n+1)
+    # if n > 1:
+    #   dy = self.get_size_y() / (n+1)
+    # else:
+    #   dy = self.get_size_y()
```

Comment:
Is this still needed?

Reply:
No, you are right. Your answer showed nicely how this approach is not needed and will unlikely be needed in the future - I just removed these lines.